impls/src/adapters/emoji.rs

```diff
@@ -1076,19 +1076,19 @@ impl EmojiSlate {
 
 		return byte as u8;
 	}
-
-	fn setExtraBits(&self, data_len: usize) -> BitVec {
+	#[warn(non_snake_case)]
```

Comment:
Why this?

Reply:
Added when I didn't want to change the function name. But after that I made the change so this `warn` is not necessary